deploy/README.md

```diff
@@ -7,28 +7,33 @@ poa-parity
 ```bash
 DEPLOYMENT_ACCOUNT_ADDRESS=0xb8988b690910913c97a090c3a6f80fad8b3a4683
 DEPLOYMENT_ACCOUNT_PRIVATE_KEY=67..14
+GAS_LIMIT=4000000
+GAS_PRICE=10
+GET_RECEIPT_INTERVAL_IN_MILLISECONDS=3000
 
 HOME_RPC_URL=https://sokol.poa.network
-HOME_OWNER_MULTISIG=0xdd2BcC1e053aBB1DfA6c1F3D6C7842f57d61440F
-HOME_UPGRADEABLE_ADMIN_VALIDATORS=0xdd2BcC1e053aBB1DfA6c1F3D6C7842f57d61440F
-HOME_UPGRADEABLE_ADMIN_BRIDGE=0xdd2BcC1e053aBB1DfA6c1F3D6C7842f57d61440F
-HOME_DAILY_LIMIT=1000000000000000000
-HOME_MAX_AMOUNT_PER_TX=100000000000000000
-HOME_MIN_AMOUNT_PER_TX=10000000000000000
-
-FOREIGN_RPC_URL=https://kovan.infura.io/mew
-FOREIGN_OWNER_MULTISIG=0xb8988b690910913c97a090c3a6f80fad8b3a4683
-FOREIGN_UPGRADEABLE_ADMIN_VALIDATORS=0xb8988b690910913c97a090c3a6f80fad8b3a4683
-FOREIGN_UPGRADEABLE_ADMIN_BRIDGE=0xb8988b690910913c97a090c3a6f80fad8b3a4683
-FOREIGN_DAILY_LIMIT=1000000000000000000
-FOREIGN_MAX_AMOUNT_PER_TX=100000000000000000
-FOREIGN_MIN_AMOUNT_PER_TX=10000000000000000
+HOME_OWNER_MULTISIG=0x
+HOME_UPGRADEABLE_ADMIN_VALIDATORS=0x
+HOME_UPGRADEABLE_ADMIN_BRIDGE=0x
+HOME_DAILY_LIMIT=30000000000000000000000000
+HOME_MAX_AMOUNT_PER_TX=1500000000000000000000000
+HOME_MIN_AMOUNT_PER_TX=500000000000000000
+HOME_REQUIRED_BLOCK_CONFIRMATIONS=1
+HOME_GAS_PRICE=1
+
+FOREIGN_RPC_URL=https://sokol.poa.network
+FOREIGN_OWNER_MULTISIG=0x
+FOREIGN_UPGRADEABLE_ADMIN_VALIDATORS=0x
+FOREIGN_UPGRADEABLE_ADMIN_BRIDGE=0x
+FOREIGN_DAILY_LIMIT=15000000000000000000000000
+FOREIGN_MAX_AMOUNT_PER_TX=750000000000000000000000
+FOREIGN_MIN_AMOUNT_PER_TX=500000000000000000
+FOREIGN_REQUIRED_BLOCK_CONFIRMATIONS=8
+FOREIGN_GAS_PRICE=10
```

Comment:
Do we want to configure `gasLimitDepositRelay` and `gasLimitWithdrawConfirm` in `.env` as well?

Reply:
we are not passing and calling those methods yet since rust is not reading them

Reply:
the same as for 'gas price' and 'required block confirmations'.
just trying to be consistent.